* Build MinGW variants with the same settings as VS variants:
	* -DOPENSSL_SYS_WIN32

diff -ru old/Configurations/10-main.conf new/Configurations/10-main.conf
--- old/Configurations/10-main.conf	2008-09-18 14:56:10.000000000 +0200
+++ new/Configurations/10-main.conf	2009-03-10 23:48:48.034829900 +0100
@@ -1400,7 +1400,7 @@
         inherit_from     => [ "BASE_unix", asm("x86_asm"),
                               sub { $disabled{shared} ? () : "x86_uplink" } ],
         cc               => "gcc",
-        cflags           => combine(picker(default => "-DL_ENDIAN -DWIN32_LEAN_AND_MEAN -DUNICODE -D_UNICODE -m32 -Wall",
+        cflags           => combine(picker(default => "-DOPENSSL_SYS_WIN32 -DL_ENDIAN -DWIN32_LEAN_AND_MEAN -DUNICODE -D_UNICODE -m32 -Wall",
                                            debug   => "-g -O0",
                                            release => "-O3 -fomit-frame-pointer"),
                                     threads("-D_MT")),
@@ -1429,7 +1429,7 @@
         # Applink is never engaged and can as well be omitted.
         inherit_from     => [ "BASE_unix", asm("x86_64_asm") ],
         cc               => "gcc",
-        cflags           => combine(picker(default => "-DL_ENDIAN -DWIN32_LEAN_AND_MEAN -DUNICODE -D_UNICODE -m64 -Wall",
+        cflags           => combine(picker(default => "-DOPENSSL_SYS_WIN32 -DL_ENDIAN -DWIN32_LEAN_AND_MEAN -DUNICODE -D_UNICODE -m64 -Wall",
                                            debug   => "-g -O0",
                                            release => "-O3"),
                                     threads("-D_MT")),
